Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### Before
+                            1. `conda uninstall seqtk`
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### Steps
+                            1. Fill a `<requirement>` tag to target a `seqtk` conda package
+                            2. Use `planemo conda_install` to install tools and set an environment
+                            3. Use `planemo test --conda_dependency_resolution ...`
+
+                        </script>
+                    </section>
+
+
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/bioconda.png" class="no-border"  height=25 />
+                            #### Bioconda
+
+                            # TODO
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/bioconda.png" class="no-border"  height=25 />
+                            #### Bioconda contributing
+
+                            # TODO
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/conda_logo.png" class="no-border"  height=25 />
+                            #### `conda build`
+
+                            # TODO
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            <img src="../images/exercise.png" alt="exercise"/>
+
+                        </script>
+                    </section>
+
                 </section>
 
+
+                <!-- PLANEMO -->
                 <section>
                     <section data-markdown>
                         <script type="text/template">
-                            ## A tool in Galaxy
+                            <img src="../images/planemo-logo.png" alt="Planemo" class="no-border" width=750/>
+
+                            > Command-line utilities to assist in building and publishing Galaxy tools.
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/planemo-logo.png" alt="Planemo" class="no-border" height=25/>
+                            ### Summary
```

Comment:
I have the feeling that this slide is not really needed with the next one. What do you think?